_includes/scripts.html

```diff
@@ -0,0 +1,25 @@
+<!-- Digital Analytics Program roll-up, see https://analytics.usa.gov for data -->
+<script id="_fed_an_ua_tag"
+  src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency={{site.dap_agency}}"></script>
+
+<!-- Google Analytics -->
+<!-- <script>
+  (function (i, s, o, g, r, a, m) {
+    i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+      (i[r].q = i[r].q || []).push(arguments)
+    }, i[r].l = 1 * new Date(); a = s.createElement(o),
+      m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+  })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+  ga('create', '{{site.google_analytics_ua}}', '{{site.url}}');
+
+  // anonymize user IPs (chops off the last IP triplet)
+  ga('set', 'anonymizeIp', true);
+
+  // forces SSL even if the page were somehow loaded over http://
+  ga('set', 'forceSSL', true);
+
+  ga('send', 'pageview');
+</script> -->
+
+{% asset uswds.min.js %}
+{% asset app.js %}
```

Comment:
The new home for scripts! Updated the asset tags to pull the correct files. The Google Analytics script is commented out because I noticed that the JA was actually getting twice since it is already added by DAP.

My question is do we still need the explicit calls to `ga` here? and if so should we be using `gas` instead? (`gas` is the DAP equivalent)